### PR TITLE
Create private command

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

**Description:**

Creates a `private` command that generates a channel for moderators to speak with a specific user (without suspending them). 

Closes #XXXXX

### CODE REVIEW

File: src/commands/private.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      //check for appropriate permissions
+      if (!message.member?.hasPermission('KICK_MEMBERS')) {
+        logger.warn(
+          `${message.author.username} did not have the correct permissions.`
+        );
+        message.channel.send(
+          'Sorry, this command is restricted to moderators.'
+        );
+        return;
+      }
```
Comment (bradtaniguchi) [inline src/commands/private.ts]: Is this the right permission check? Maybe we can use the `MOD_ROLE` environment variable check instead?

File: src/commands/private.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      //logging embed
+      const restrictEmbed = new MessageEmbed()
+        .setColor('#FF0000')
+        .setTitle(`Private Channel Created`)
+        .addFields({
+          name: 'What happened?',
+          value: `${mod} has created a private discussion channel for **${user.user.username}** (${user}).`
+        })
+        .setFooter('Please remember to follow our rules!');
+      modChannel.send(restrictEmbed);
```
Comment (bradtaniguchi) [inline src/commands/private.ts]: Is this correct? Seems a lot like the normal suspension flow.

**Review by bradtaniguchi — COMMENTED** [review]
Is this flow the same as the normal suspension flow, except we don't change the user's role? Might be worth thinking about refactoring these 2 workflows into 1

File: src/commands/private.ts (hunk 1 above)
Comment (naomi-lgbt) [reply src/commands/private.ts]: That's a good approach, actually.

File: src/commands/private.ts (hunk 2 above)
Comment (naomi-lgbt) [reply src/commands/private.ts]: Oh I should change the wording on the footer - but this is our mod-log embed and the action should still be logged I think 🤔

**Review by bradtaniguchi — APPROVED** [review]
LGTM :+1:

**Pull request merged**